packages/terra-core-docs/src/terra-dev-site/doc/image/example/DecorativeImage/DecorativeImageBar.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { DecorativeImage } from 'terra-image';
+import decorativeBar from '../../assets/decorative.png';
```

Comment:
nit: could we rename the png to `decorative-bar.png`?

Reply:
[updated](https://github.com/cerner/terra-core/pull/3585/commits/a72252e81ed3e48fbebc3ff03878cf6f428b0261#diff-9fab70ff6d883d041e5b13780ab2f60911455f4c15d86ea150e124253c2820fc)